--- project/tmp/src/logic/1.agilogic	2025-12-22 14:24:39.537119007 +0000
+++ /tmp/1.agilogic.modified	2025-12-22 14:50:28.215517001 +0000
@@ -355,7 +355,7 @@
     open.dialogue();
     set.string(s2, m31);
     display(16, 0, m31);
-    get.string(s2, m26, 16, 1, 18);
+    get.string(s2, m26, 16, 30, 18);
     parse(s2);
   
     if (!f2) {
@@ -364,7 +364,7 @@
       set.cursor.char(m33);
       prevent.input();
     
-      if (said("astral", "bodies")) {
+      if (said("bodies", "astral")) {
         display(18, 13, m27);
       
         if (v50 > 0) {
@@ -759,8 +759,8 @@
       status.line.off();
       prevent.input();
       text.screen();
-      display(2, 13, m19);
-      display(3, 11, m20);
+      display(2, 2, m19);
+      display(3, 2, m20);
       display(5, 2, m21);
       display(8, 2, m22);
       display(9, 3, m23);
