--- project/tmp/src/logic/10.agilogic	2025-12-23 14:45:29.185532000 +0000
+++ /tmp/10.agilogic.modified	2025-12-25 14:26:26.300825008 +0000
@@ -303,9 +303,6 @@
     }
   
   Address1656:
-  
-    if (v31 == 7) {
-    }
   }
 
 Address1663:
@@ -553,15 +550,15 @@
         v64 = 104;
         set(f32);
         text.screen();
-        display(5, 6, m4);
-      
+        display(5, 7, m4);
+
         if (f79) {
-          display(11, 11, m42);
+          display(11, 12, m42);
         } else {
-          display(7, 9, m5);
-          display(9, 7, m6);
-          display(11, 8, m8);
-          display(12, 13, m9);
+          display(7, 14, m5);
+          display(9, 12, m6);
+          display(11, 7, m8);
+          display(12, 15, m9);
         }
       } else {
         print(m3);
@@ -587,7 +584,7 @@
 
 Address828:
 
-  if (said("press", "autonav") || said("autonav", "a") || said("press", "autonav", "button")) {
+  if (said("press", "autonav") || said("autonav", "a") || said("press", "button", "autonav")) {
     if (f188) {
       if (!f81) {
         set(f81);
@@ -613,7 +610,7 @@
 
 Address912:
 
-  if (said("press", "power") || said("power", "a") || said("press", "power", "button")) {
+  if (said("press", "power") || said("power", "a") || said("press", "button", "power")) {
     if (!f188) {
       set(f188);
       prevent.input();
@@ -722,7 +719,7 @@
 
 Address1230:
 
-  if (said("get down", "belt")) {
+  if (said("get down", "belt") || said("open", "belt") || said("unlock", "belt")) {
     if (f44) {
       if (v1 == 12) {
         print(m46);
@@ -751,7 +748,7 @@
 
 Address1315:
 
-  if (said("press", "don't touch") || said("press", "don't touch", "button")) {
+  if (said("press", "don't touch") || said("press", "button", "don't touch")) {
     if (f79) {
       set(f71);
       sound(14, f162);
