--- project/tmp/src/logic/6.agilogic	2025-12-22 14:24:39.643119007 +0000
+++ /tmp/6.agilogic.modified	2025-12-22 14:51:51.590971012 +0000
@@ -204,7 +204,7 @@
 Address528:
 
   if (!f31) {
-    if (said("press", "open", "bay door") || said("press", "open", "bay door", "button") || said("press", "bay door", "open", "button") || said("press", "bay door", "open") || said("press", "open")) {
+    if (said("press", "open", "bay door") || said("press", "button", "open", "bay door") || said("press", "button", "bay door", "open") || said("press", "bay door", "open") || said("press", "open")) {
       if (v52 == 0) {
         if (obj.in.box(o0, 86, 125, 134, 136)) {
           set(f31);
@@ -221,7 +221,7 @@
   
   Address620:
   
-    if (said("press", "close", "bay door") || said("press", "close")) {
+    if (said("press", "close", "bay door") || said("press", "button", "close", "bay door") || said("press", "button", "bay door", "close") || said("press", "bay door", "close") || said("press", "close")) {
       if (v52 == 1) {
         if (!f54) {
           if (obj.in.box(o0, 86, 125, 134, 136)) {
