--- /home/devuser/agivrit/project/tmp/src/logic/67.agilogic	2025-12-07 00:05:53.961529003 +0000
+++ /tmp/67.agilogic.modified	2025-12-12 21:17:12.214677002 +0000
@@ -12,6 +12,7 @@
   load.view(204);
   load.view(205);
   load.view(206);
+  load.view(254);
   load.sound(60);
   animate.obj(o1);
   set.view(o1, 200);
@@ -21,7 +22,7 @@
   set.view(o2, 201);
   set.loop(o2, 0);
   set.cel(o2, 0);
-  position(o2, 45, 146);
+  position(o2, 47, 161);
   set.priority(o2, 15);
   animate.obj(o3);
   animate.obj(o4);
@@ -71,7 +72,7 @@
   v33 = 1;
   set.loop(o2, 0);
   set.cel(o2, 0);
-  position(o2, 45, 146);
+  position(o2, 47, 161);
   position(o1, 23, 23);
   draw.pic(v0);
   show.pic();
@@ -108,7 +109,7 @@
       set.cel(o2, 0);
       stop.cycling(o2);
       draw(o1);
-      move.obj(o1, 23, 137, 6, f30);
+      move.obj(o1, 23, 152, 6, f30);
     }
   
     if (f30) {
@@ -131,7 +132,7 @@
       reset(f32);
     
       if (v32 > 0) {
-        position(o1, 23, 137);
+        position(o1, 23, 152);
         draw(o1);
         move.obj(o1, 23, 23, 6, f30);
       }
@@ -168,10 +169,15 @@
       erase(o8);
       erase(o9);
     
-      if (v33 == 4) {
+      if (v33 == 5) {
         set(f168);
       }
     
+      if (v33 == 4) {
+        v33 = 5;
+        set(f40);
+      }
+    
       if (v33 == 3) {
         v33 = 4;
         set(f40);
@@ -197,27 +203,27 @@
       set.view(o3, 202);
       set.loop(o3, 0);
       set.cel(o3, 0);
-      position(o3, 45, 41);
+      position(o3, 66, 41);
       set.view(o4, 203);
       set.cel(o4, 0);
-      position(o4, 94, 41);
+      position(o4, 65, 56);
       set.view(o5, 204);
       set.loop(o5, 0);
       set.cel(o5, 0);
       set.view(o6, 205);
       set.cel(o6, 0);
-      position(o6, 66, 79);
+      position(o6, 78, 94);
       set.view(o7, 206);
       set.loop(o7, 0);
       set.cel(o7, 0);
       random(1, 100, v126);
     
       if (v126 > 50) {
-        position(o5, 46, 98);
-        position(o7, 44, 61);
+        position(o5, 53, 113);
+        position(o7, 55, 76);
       } else {
-        position(o5, 46, 61);
-        position(o7, 41, 98);
+        position(o5, 53, 76);
+        position(o7, 55, 113);
       }
       draw(o3);
       draw(o4);
@@ -239,30 +245,30 @@
       set.view(o3, 202);
       set.loop(o3, 3);
       set.cel(o3, 0);
-      position(o3, 68, 32);
+      position(o3, 63, 32);
       stop.cycling(o3);
       set.view(o4, 202);
       set.loop(o4, 3);
       set.cel(o4, 1);
-      position(o4, 48, 44);
+      position(o4, 65, 43);
       stop.cycling(o4);
       set.view(o5, 202);
       set.loop(o5, 3);
       set.cel(o5, 2);
-      position(o5, 63, 56);
+      position(o5, 60, 56);
       stop.cycling(o5);
       set.view(o6, 203);
-      position(o6, 75, 73);
+      position(o6, 65, 73);
       set.view(o7, 206);
       set.loop(o7, 1);
       set.cel(o7, 0);
-      position(o7, 38, 92);
+      position(o7, 53, 92);
       set.view(o8, 205);
-      position(o8, 71, 108);
+      position(o8, 78, 108);
       set.view(o9, 204);
       set.loop(o9, 1);
       set.cel(o9, 0);
-      position(o9, 54, 125);
+      position(o9, 56, 125);
       draw(o3);
       draw(o4);
       draw(o5);
@@ -284,21 +290,21 @@
       reset(f40);
       set.view(o3, 202);
       set.loop(o3, 1);
-      position(o3, 41, 52);
+      position(o3, 65, 52);
       set.view(o4, 203);
-      position(o4, 109, 52);
+      position(o4, 65, 67);
       set.view(o5, 206);
       set.loop(o5, 0);
       set.cel(o5, 0);
-      position(o5, 46, 76);
+      position(o5, 55, 91);
       set.view(o7, 206);
       set.loop(o7, 2);
       set.cel(o7, 0);
-      position(o7, 45, 95);
+      position(o7, 58, 109);
       set.view(o8, 204);
       set.loop(o8, 2);
       set.cel(o8, 0);
-      position(o8, 47, 111);
+      position(o8, 55, 126);
       draw(o3);
       draw(o4);
       draw(o5);
@@ -320,13 +326,13 @@
       reset(f40);
       set.view(o3, 202);
       set.loop(o3, 2);
-      position(o3, 45, 61);
+      position(o3, 64, 61);
       set.view(o4, 203);
-      position(o4, 95, 61);
+      position(o4, 65, 76);
       set.view(o5, 204);
       set.loop(o5, 0);
       set.cel(o5, 0);
-      position(o5, 47, 82);
+      position(o5, 53, 97);
       draw(o3);
       draw(o4);
       draw(o5);
@@ -337,6 +343,31 @@
       reset(f35);
       v34 = 40;
     }
+  
+    if (f40 && (v33 == 5)) {
+      reset(f40);
+      set.view(o3, 254);
+      set.loop(o3, 0);
+      set.cel(o3, 0);
+      position(o3, 66, 56);
+      stop.cycling(o3);
+      set.view(o4, 203);
+      position(o4, 65, 72);
+      stop.cycling(o4);
+      set.view(o5, 254);
+      set.loop(o5, 1);
+      set.cel(o5, 0);
+      position(o5, 60, 111);
+      draw(o3);
+      draw(o4);
+      draw(o5);
+      end.of.loop(o5, f35);
+    }
+  
+    if ((v33 == 5) && f35) {
+      reset(f35);
+      v34 = 40;
+    }
   }
 }
 
