--- project/tmp/src/logic/67.agilogic	2025-12-07 00:05:53.961529003 +0000
+++ translations/sq1/logic/67.agilogic	2025-12-11 01:55:48.838831014 +0000
@@ -21,7 +21,7 @@
   set.view(o2, 201);
   set.loop(o2, 0);
   set.cel(o2, 0);
-  position(o2, 45, 146);
+  position(o2, 47, 161);
   set.priority(o2, 15);
   animate.obj(o3);
   animate.obj(o4);
@@ -71,7 +71,7 @@
   v33 = 1;
   set.loop(o2, 0);
   set.cel(o2, 0);
-  position(o2, 45, 146);
+  position(o2, 47, 161);
   position(o1, 23, 23);
   draw.pic(v0);
   show.pic();
@@ -108,7 +108,7 @@
       set.cel(o2, 0);
       stop.cycling(o2);
       draw(o1);
-      move.obj(o1, 23, 137, 6, f30);
+      move.obj(o1, 23, 152, 6, f30);
     }
   
     if (f30) {
@@ -131,7 +131,7 @@
       reset(f32);
     
       if (v32 > 0) {
-        position(o1, 23, 137);
+        position(o1, 23, 152);
         draw(o1);
         move.obj(o1, 23, 23, 6, f30);
       }
