--- project/tmp/src/logic/75.agilogic	2025-12-23 14:45:29.303532000 +0000
+++ /tmp/75.agilogic.modified	2025-12-26 21:17:41.732137001 +0000
@@ -16,7 +16,7 @@
   set.view(o9, 92);
   set.cel(o9, 0);
   ignore.objs(o9);
-  position(o9, 52, 154);
+  position(o9, 52, 154);
   set.priority(o9, 4);
   stop.cycling(o9);
   animate.obj(o8);
@@ -48,39 +48,39 @@
   stop.cycling(o9);
 
   if ((v26 == 2) && (v20 == 0)) {
-    display(18, 13, m49);
-    display(19, 13, m49);
-    display(20, 13, m49);
-    display(21, 13, m49);
+    display(18, 12, m49);
+    display(19, 12, m49);
+    display(20, 12, m49);
+    display(21, 12, m49);
   } else {
-    display(16, 13, m49);
-    display(17, 13, m49);
-    display(18, 13, m49);
-    display(19, 13, m49);
+    display(16, 12, m49);
+    display(17, 12, m49);
+    display(18, 12, m49);
+    display(19, 12, m49);
   }
 
 Address163:
 
   if ((v26 == 2) && (v20 == 0)) {
     clear.lines(1, 2, 0);
-    display(1, 15, m59);
-    display(2, 0, m60);
+    display(1, 14, m59);
+    display(2, 0, m60);
   } else {
     clear.lines(22, 24, 0);
-    display(23, 15, m59);
-    display(24, 0, m60);
+    display(23, 14, m59);
+    display(24, 0, m60);
   }
 
 Address200:
 
   if ((v26 == 2) && (v20 == 0)) {
-    display(19, 19, m5);
-    display(20, 19, m6);
-    display(21, 19, m7);
+    display(19, 12, m5);
+    display(20, 12, m6);
+    display(21, 12, m7);
   } else {
-    display(17, 19, m5);
-    display(18, 19, m6);
-    display(19, 19, m7);
+    display(17, 12, m5);
+    display(18, 12, m6);
+    display(19, 12, m7);
   }
 
 Address237:
@@ -98,207 +98,207 @@
 
   if (!f128) {
   Address433:
-  
+
     if (!f130) {
     Address523:
-    
+
       if (!f2 || f4) {
       Address617:
-      
+
         if (f124 && !f95) {
           set(f95);
         }
-      
+
       Address628:
         v30--;
-      
+
         if (v30 == 1) {
           if (v35 == 4) {
             v35 = 3;
             v34 = 100;
-          
+
             if ((v26 == 2) && (v20 == 0)) {
-              display(18, 13, m13);
-              display(19, 13, m49);
-              display(20, 13, m14);
-              display(21, 13, m49);
+              display(18, 12, m13);
+              display(19, 12, m49);
+              display(20, 12, m14);
+              display(21, 12, m49);
             } else {
-              display(16, 13, m13);
-              display(17, 13, m49);
-              display(18, 13, m14);
-              display(19, 13, m49);
+              display(16, 12, m13);
+              display(17, 12, m49);
+              display(18, 12, m14);
+              display(19, 12, m49);
             }
           }
-        
+
         Address695:
-        
+
           if (v35 == 2) {
             v35 = 4;
-          
+
             if ((v26 == 2) && (v20 == 0)) {
-              display(18, 13, m61);
-              display(19, 13, m62);
-              display(20, 13, m63);
-              display(21, 13, m64);
+              display(18, 12, m61);
+              display(19, 12, m62);
+              display(20, 12, m63);
+              display(21, 12, m64);
             } else {
-              display(16, 13, m61);
-              display(17, 13, m62);
-              display(18, 13, m63);
-              display(19, 13, m64);
+              display(16, 12, m61);
+              display(17, 12, m62);
+              display(18, 12, m63);
+              display(19, 12, m64);
             }
-          
+
           Address750:
             v30 = 100;
           }
-        
+
         Address753:
-        
+
           if (v35 == 1) {
             v35 = 2;
-          
+
             if ((v26 == 2) && (v20 == 0)) {
-              display(18, 20, m48);
-              display(19, 19, m8);
-              display(20, 19, m9);
-              display(21, 19, m10);
+              display(18, 12, m48);
+              display(19, 12, m8);
+              display(20, 12, m9);
+              display(21, 12, m10);
             } else {
-              display(16, 20, m48);
-              display(17, 19, m8);
-              display(18, 19, m9);
-              display(19, 19, m10);
+              display(16, 12, m48);
+              display(17, 12, m8);
+              display(18, 12, m9);
+              display(19, 12, m10);
             }
-          
+
           Address808:
             v34 = 100;
           }
         }
-      
+
       Address811:
         v34--;
-      
+
         if (v34 == 1) {
           if (v35 == 1) {
-            position(o9, 52, 154);
+            position(o9, 52, 154);
             set.cel(o9, 1);
             stop.cycling(o9);
             v30 = 2;
           }
-        
+
         Address839:
-        
+
           if (v35 == 3) {
             v35 = 1;
             v34 = 100;
-          
+
             if ((v26 == 2) && (v20 == 0)) {
-              display(18, 13, m49);
-              display(19, 13, m49);
-              display(20, 13, m49);
+              display(18, 12, m49);
+              display(19, 12, m49);
+              display(20, 12, m49);
             } else {
-              display(16, 13, m49);
-              display(17, 13, m49);
-              display(18, 13, m49);
+              display(16, 12, m49);
+              display(17, 12, m49);
+              display(18, 12, m49);
             }
-          
+
           Address889:
             set.cel(o9, 0);
-            position(o9, 52, 154);
+            position(o9, 52, 154);
             draw(o9);
             stop.cycling(o9);
-          
+
             if ((v26 == 2) && (v20 == 0)) {
-              display(19, 19, m5);
-              display(20, 19, m6);
-              display(21, 19, m7);
+              display(19, 12, m5);
+              display(20, 12, m6);
+              display(21, 12, m7);
             } else {
-              display(17, 19, m5);
-              display(18, 19, m6);
-              display(19, 19, m7);
+              display(17, 12, m5);
+              display(18, 12, m6);
+              display(19, 12, m7);
             }
           }
-        
+
         Address937:
-        
+
           if (v35 == 2) {
             erase(o9);
-          
+
             if ((v26 == 2) && (v20 == 0)) {
-              display(18, 20, m48);
+              display(18, 12, m48);
             } else {
-              display(16, 20, m48);
+              display(16, 12, m48);
             }
-          
+
           Address967:
             v30 = 2;
-          
+
             if ((v26 == 2) && (v20 == 0)) {
-              display(18, 13, m49);
-              display(19, 13, m49);
-              display(20, 13, m49);
-              display(21, 13, m49);
+              display(18, 12, m49);
+              display(19, 12, m49);
+              display(20, 12, m49);
+              display(21, 12, m49);
             } else {
-              display(16, 13, m49);
-              display(17, 13, m49);
-              display(18, 13, m49);
-              display(19, 13, m49);
+              display(16, 12, m49);
+              display(17, 12, m49);
+              display(18, 12, m49);
+              display(19, 12, m49);
             }
           }
         }
-      
+
       Address1015:
-      
+
         if (v35 == 6) {
           v32--;
-        
+
           if (v32 == 1) {
             v32 = 250;
             v33--;
-          
+
             if (v33 == 1) {
               v32 = 0;
               v33 = 3;
-            
+
               if ((v26 == 2) && (v20 == 0)) {
-                display(18, 13, m49);
-                display(19, 13, m49);
-                display(20, 13, m49);
-                display(21, 13, m49);
+                display(18, 12, m49);
+                display(19, 12, m49);
+                display(20, 12, m49);
+                display(21, 12, m49);
               } else {
-                display(16, 13, m49);
-                display(17, 13, m49);
-                display(18, 13, m49);
-                display(19, 13, m49);
+                display(16, 12, m49);
+                display(17, 12, m49);
+                display(18, 12, m49);
+                display(19, 12, m49);
               }
-            
+
             Address1094:
               v34 = 9;
               v35 = 3;
             }
           }
         }
-      
+
       Address1100:
-      
+
         if (f233) {
           reset(f233);
-        
+
           if ((v26 == 2) && (v20 == 0)) {
-            display(18, 13, m49);
-            display(19, 13, m49);
-            display(20, 13, m49);
-            display(21, 13, m49);
+            display(18, 12, m49);
+            display(19, 12, m49);
+            display(20, 12, m49);
+            display(21, 12, m49);
           } else {
-            display(16, 13, m49);
-            display(17, 13, m49);
-            display(18, 13, m49);
-            display(19, 13, m49);
+            display(16, 12, m49);
+            display(17, 12, m49);
+            display(18, 12, m49);
+            display(19, 12, m49);
           }
-        
+
         Address1153:
-        
+
           if ((v26 == 2) && (v20 == 0)) {
           Address1170:
-            display(24, 10, m1);
+            display(24, 12, m1);
             draw(o8);
             end.of.loop(o8, f30);
           } else {
@@ -393,13 +393,13 @@
           if (said("holy", "asshole")) {
             v121 = 134;
             clear.lines(23, 24, 0);
-            display(23, 0, m3);
-            display(24, 0, m4);
+            display(23, 0, m3);
+            display(24, 0, m4);
             get.num(m2, v122);
             *v121 = v122;
             clear.lines(23, 24, 0);
-            display(23, 15, m59);
-            display(24, 0, m60);
+            display(23, 14, m59);
+            display(24, 0, m60);
           }
           goto(Address605);
         }
